### PR TITLE
add https request url for calendar service call

### PR DESCRIPTION
**Pull request by yinan-liu**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java [hunk 1]
```diff
@@ -200,8 +200,14 @@ private void loadWindowsContent(RequestBuilder.Method method) {
...
         String user = LoginModel.getInstance().getLogin();
-        String requestUrl = "http://" + host + ":" + SchedulerConfig.get().getCalendarServerPort() +
-            "/calendar-service/private-urls/" + user + "/";
+        String requestUrl = null;
+
+        if ("http".equals(SchedulerConfig.get().getCalendarServerProtocol())) {
+            requestUrl = "http://" + host + ":" + SchedulerConfig.get().getCalendarServerPort() +
+                "/calendar-service/private-urls/" + user + "/";
+        } else {
+            requestUrl = "https://" + host + "/calendar-service/private-urls/" + user + "/";
```
Comment (marcocast) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java]: The following part could be put outside the if  : 

requestUrl  +=  "/calendar-service/private-urls/" + user + "/";

Comment (marcocast) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java]: Can this be unit tested?

Comment (yinan-liu) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java]: i think yes, i can add a test later

Comment (yinan-liu) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/CalendarInfoWindow.java]: After having checked, i dont think i can do a junit test on it, as SchedulerConfig.get() is a static class.

**Pull request merged**